schedule/sle-micro/raw_image.yaml

```diff
@@ -3,6 +3,15 @@ description:    >
     Maintainer: jalausuch@suse.com, qa-c@suse.de.
     SUSE Linux Enterprise Micro tests
 conditional_schedule:
+  boot:
+    ARCH:
+      's390x':
+        - installation/bootloader_start
+        - boot/boot_to_desktop
```

Comment:
why `microos/disk_boot` can be used here?
i am thinking that if we need to do some modification specific to microos in the future we will have to do it first in `microos/disk_boot` and subsequently to the `boot/boot_to_desktop`.

Reply:
it doesn't boot, https://openqa.suse.de/tests/6299774#step/disk_boot/1
we need to do some work to change it as it interacts with the host using virsth commands...

Reply:
but this link compared to the VR looks the same exact issue.

Reply:
no, that link is before this PR. Applying this PR I see the system booting and trying to get config from ignition: https://openqa.suse.de/tests/6293548#step/boot_to_desktop/7
but for some reason it fails..

Reply:
hm...I am just wondering if the bootloader abstraction is needed. Otherwise, I would go for `bootloader_zkvm.pm` directly. But that is neat-picking and feel free to ignore.